e2e/commands_test.go

```diff
@@ -74,6 +74,117 @@ func TestRenderFormatters(t *testing.T) {
 	golden.Assert(t, result.Stdout(), "expected-yaml-render.golden")
 }
 
+func TestInitWarningEnvFile(t *testing.T) {
+	cmd, cleanup := dockerCli.createTestCmd()
+	defer cleanup()
+
+	composeData := `version: "3.2"
+services:
+  nginx:
+    image: nginx:latest
+    command: nginx $NGINX_ARGS ${NGINX_DRY_RUN}
+    env_file: myenv1.env`
+	meta := `# Version of the application
+version: 0.1.0
+# Name of the application
+name: app-test
+# A short description of the application
+description: my cool app
+# List of application maintainers with name and email for each
+maintainers:
+  - name: dev1
+    email: 
+  - name: dev2
+    email: dev2@example.com
+`
+	envData := "# some comment\nNGINX_DRY_RUN=-t"
+	tmpDir := fs.NewDir(t, "app_input",
+		fs.WithFile(internal.ComposeFileName, composeData),
+		fs.WithFile(".env", envData),
+		fs.WithFile("myenv1.env", ""),
+	)
+	defer tmpDir.Remove()
+
+	testAppName := "app-test"
+	dirName := internal.DirNameFromAppName(testAppName)
+
+	cmd.Dir = tmpDir.Path()
+	cmd.Command = dockerCli.Command("app",
+		"init", testAppName,
+		"--compose-file", tmpDir.Join(internal.ComposeFileName),
+		"--description", "my cool app",
+		"--maintainer", "dev1",
+		"--maintainer", "dev2:dev2@example.com")
+	stdOut := icmd.RunCmd(cmd).Assert(t, icmd.Success).Combined()
+	golden.Assert(t, stdOut, "init-output-warning-envfile.golden")
+
+	manifest := fs.Expected(
+		t,
+		fs.WithMode(0755),
+		fs.WithFile(internal.MetadataFileName, meta, fs.WithMode(0644)),
+		fs.WithFile(internal.ComposeFileName, composeData, fs.WithMode(0644)),
+		fs.WithFile(internal.ParametersFileName, "NGINX_ARGS: FILL ME\nNGINX_DRY_RUN: -t\n", fs.WithMode(0644)),
+	)
+	assert.Assert(t, fs.Equal(tmpDir.Join(dirName), manifest))
+}
+
+func TestInitWarningEnvFiles(t *testing.T) {
```

Comment:
What are you testing here, except 2 env files? I think that you can remove either the first or the second test.